baseframe/forms/validators.py

```diff
@@ -42,7 +59,7 @@ def __call__(self, form, field):
                 raise StopValidation(self.message)
 
 
-class OptionalIfNot(OptionalIf):
+class RequiredIf(OptionalIf):
     """
     Validator that makes this field optional if the value of some other field is false.
     """
```

Comment:
From the description and code, this it not a `RequiredIf`. Rename back to `OptionalIfNot`. We need a separate `RequiredIf`.